tests/unit/phpDocumentor/Guides/Document/HtmlTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Document;
+
+use phpDocumentor\Guides\ContentType;
+use phpDocumentor\Path;
+
+class HtmlTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetContentType()
```

Comment:
In the other tests you use `@covers` annotations but not in this; is this on purpose?